content/pt-br/contributor-ladder/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Escala de Contribuição
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-pen-square'></i>
+---
+
+Olá! 👋 Obrigado pelo seu interesse em contribuir com o projeto CNCF Glossário Cloud Native. Seja contribuindo com novos termos, ajudando a localizar o glossário em seu idioma nativo ou ajudando outras pessoas a começar, há muitas maneiras de se tornar um membro ativo desta comunidade. Este documento descreve as diferentes funções do colaborador no projeto e as responsabilidades e privilégios que as acompanham.
+
+## 1. Contribuidores
+
+O glossário é para todos. Qualquer pessoa pode se tornar um contribuidor simplesmente contribuindo com o projeto.
+É esperado que todos os contribuidores sigam o [código de conduta da CNCF](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+Existem várias formas de você contribuir com o projeto, incluindo:
+
+- **Contribuidores de conteúdo**: qualquer pessoa que melhore os termos existentes ou contribua com novos,
```

Comment:
I suggest to drop the comma at the end of each point - even though in the english version there is a comma there

Reply:
Hmm, I was [searching](https://ead.pucpr.br/blog/quando-usar-ponto-virgula), and looks like in Portuguese we can use the `;`, what do you think?